pkgs/applications/office/qnotero/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, python3, python3Packages, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "qnotero-${version}";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "smathot";
+    repo = "qnotero";
+    rev = "release/1.0.0";
+    name = "qnotero-1.0.0-src";
+    sha256 = "1d5a9k1llzn9q1qv1bfwc7gfflabh4riplz9jj0hf04b279y1bj0";
+  };
+
+  buildInputs = [ python3 python3Packages.pyqt4 makeWrapper ];
+
+  buildPhase = "python3 ./setup.py build";
+
+  installPhase = ''
+      substituteInPlace ./setup.py \
+        --replace "/usr/share" "$out/usr/share"
+
+      substituteInPlace ./libqnotero/_themes/default.py \
+        --replace "/usr/share" "$out/usr/share"
```

Comment:
Generally it's preferable to use pre/postPatch, otherwise patches get ignored silently when overriding the patchPhase.